src/Nethermind/Nethermind.AuRa/Validators/ListValidator.cs

```diff
@@ -47,7 +47,10 @@ public ListValidator(AuRaParameters.Validator validator)
 
         public void Initialize(Block block, TransactionProcessor transactionProcessor) { }
 
-        public bool IsValidSealer(Address address) => _validatorAddresses.Contains(address);
+        public bool IsValidSealer(Address address) => _validators.Contains(address);
+        public int MinSealersForFinalization => _validators.MinSealersForFinalization();
+        void IAuRaValidator.SetFinalizationManager(IBlockFinalizationManager finalizationManager) { }
```

Comment:
why does it do nothing? maybe a comment?

Reply:
ListValidator doesn't change its behaviour/state based on Finalization of blocks, only Multi and COntract validator do.

Reply:
cool, so maybe just adding this comment would be ok